esmvaltool/diag_scripts/examples/my_little_diagnostic.py

```diff
@@ -54,16 +101,20 @@ def plot_time_series(cfg):
     out_path = 'esmvaltool_users/valeriu/'   # edit as per need
```

Comment:
Please write the output to the directory `cfg['work_dir']` or `cfg['plot_dir']`, writing just anywhere in the file system makes a mess of things.